docs/users-guide/08-dashboard-filters.md

```diff
@@ -4,84 +4,97 @@
 
 ![Search options](images/dashboard-filters/search-options.png)
 
+### Linking filters
+
+You can also **link filters** so that a child filter knows to limit its choices based on the activation of a parent filter.
+
+Say you have two filters, one to filter by state, the other to filter by city. You can link the city filter to the state filter so that when someone filter for California, the city filter will only show cities in California. In this case, state is the parent filter, and city is the child filter.
+
+To link filters, you'll need to set up this parent-child relationship. And you set up this relationship through the child filter. In the above scenario, with a state and city filter, we'd edit the child filter, city, by clicking on the **gears** icon on the city filter. From the filter sidebar on the right, select the **Linked filters** tab. 
+
+![Linked filters](images/dashboard-filters/linked-filter.png)
+
+Here you can limit the current filter's choices. If you toggle on one of these dashboard filters, selecting a value for that filter will limit the available choices for this filter. In this case, we toggle on the state filter (the parent), to limit the choices for the city filter. When states are selected, the city filter will limit its choices to cities in those states. Click **Done**, then **Save** to save the dashboard.
+
 ### Best practices
 
 Here are a few tips to get the most out of dashboard filters:
 
-- Try to keep the number of filters you add to a dashboard to two or three. This will make it easier for your teammates to quickly and easily understand what options are available to them when viewing your dashboard.
-- While you can add dashboard filters to a dashboard that already has a bunch of cards in it, it can be easier to start a new dashboard and think about what filters you intend to add to it, and then make sure that you only put cards in that dashboard that can be used with the filters.
+- **Limit filters to 3 or fewer**. Try to keep the number of filters you add to a dashboard to two or three. This will make it easier for your teammates to quickly understand what options are available to them when viewing your dashboard.
+- **Start with a new dashboard**. While you can add dashboard filters to an existing dashboard with a bunch of cards in it, it can be easier to start a new dashboard and think about what filters you intend to add to it, and then make sure that you only put cards in that dashboard that can be used with the filters. You can also duplicate an existing dashboard, and pare down the number of cards.
+- **Link filters** so people don't have to sift through irrelevant filter options (like cities not in a filtered state).
+- **Use a chart or table to update a filter**. You can [set up a question to update a filter on click](interactive-dashboards.md). 
```

Comment:
> Use a chart or table to update a filter

Is this really a "best practice?" We mention this capability elsewhere on this page, so I don't think we need to force it in here.